### PR TITLE
Created CORTX Troubleshooting Guide

### PR DESCRIPTION
**Pull request by Saumya-Sunder**
Created CORTX Troubleshooting Guide

Signed-off-by: Saumya Sunder <saumya.sunder@seagate.com>

### CODE REVIEW

File: Troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+We've listed all the known issues with the CORTX software and the steps to resolve them below. Please refer to our [support documentation](SUPPORT.md) to report new issues or want us to document steps to resolve an issue.
```
Comment (johnbent) [inline Troubleshooting.md:1]: 'want us to document . . . ' is not grammatically correct.

Comment (johnbent) [reply Troubleshooting.md:1]: Maybe you meant 'or if you' before 'want'?

File: Troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
+We've listed all the known issues with the CORTX software and the steps to resolve them below. Please refer to our [support documentation](SUPPORT.md) to report new issues or want us to document steps to resolve an issue. 
+
+# Known Issues
+
+<details>
+<summary>Firmware bundle upload and update functionality not working</summary>
```
Comment (johnbent) [inline Troubleshooting.md:6]: I have no idea what this means.  Do you?

File: Troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+### Symptoms and causes
+  
+  1. Failed to fetch update status.
+  2. Failed to upload firmware bundle file.
+  3. Failed to update the firmware.
+
+### Resolution
+
+  1. Failed to fetch update status.
+    a. Refresh CMP UI after 30 secs.
```
Comment (johnbent) [inline Troubleshooting.md:18]: I have no idea what any of this means.  Nor will the reader.  This is not a useful document yet.

File: Troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+ </p>
+ </details>
+  
+<details>
+<summary>CMP UI is not working</summary>
+<p>
+
+### Symptoms and causes
+
+1. URL is incorrect.
```
Comment (johnbent) [inline Troubleshooting.md:57]: What is this?  What URL?  This document needs to be much more precise.  Something like, "During X, you may see some error Y.  In such a case, here is what to do."

Currently this document is very far from being useful.

Perhaps asking for this document was not a good idea.  All of these steps should be in the individual quick start guides.  I think that is the biggest problem.  We have a bunch of quick start guides and these instructions are all for different ones of them.  I think this whole document is a mistake.  

On the basis of what I've seen so far, I don't think this document will be useful.

**Review by johnbent — COMMENTED** [review]
I don't know whether this document can be useful.  I think that all of this stuff just needs to go directly into the relevant quick start guides.  It is not useful to be aggregated into one single document.  Right now this document is just massively confusing.

Also . . . when we add new files, we need to also add links to them from other existing files.  But that seems to have been forgotten this time.  Please add a link to this from somewhere else.  Otherwise there is zero value to a new document if the reader will never find it.

Having said that, I'm not sure this document can be useful anyway....

**Comment by Saumya-Sunder** [comment]
Closing the PR as the Troubleshooting Guide is not structured correctly.

File: Troubleshooting.md (hunk 2 above)
Comment (Saumya-Sunder) [reply Troubleshooting.md:6]: Fixed it.

**Comment by mukul-seagate11** [comment]
@Saumya-Sunder, do resolve conflicts

**Comment by mukul-seagate11** [comment]
Closing the PR as it has conflicts as well as Saumya had left the org

**Pull request closed without merging**